registration/views.py

```diff
@@ -347,7 +359,7 @@ def process_registration_configuration(self):
                 self.page_explanatory_texts[step] = explanatory_text
                 self.page_fieldsets[step] = fieldsets
                 self.form_list[unicode(step)] = forms.register_form_generator(
-                    conf=page_conf)
+                    conf=page_conf, license_id_formats=license_id_formats)
```

Comment:
@mabuaisha Can't we just set the license id formats somehow within page_conf instead of passing it as a separate parameter? Is there any reason why we should have it separately?